src/main/java/org/springframework/cloud/dataflow/module/deployer/cloudfoundry/CloudFoundryModuleDeployerProperties.java

```diff
@@ -82,6 +82,16 @@
 	 */
 	private String buildpack = "https://github.com/cloudfoundry/java-buildpack.git#master";
 
+	/**
+	 * The amount of memory (MB) to allocate, if not overridden per-module.
+	 */
+	private int memory = 1024;
+
+	/**
+	 * The amount of disk space (MB) to allocate, if not overridden per-module.
+	 */
+	private int disk;
```

Comment:
We can provide a default value here as well or at least pass a null if zero. Not sure what's the effect of passing the zero value downstream.